standard/clause_0_front_material.adoc

```diff
@@ -18,19 +18,19 @@ Given the intended use case of downloading data to a browser, there is no functi
 [abstract]
 == Abstract
 
-Based on JavaScript Object Notation (JSON), CoverageJSON is a format for publishing spatiotemporal data to the Web. The primary design goals are simplicity, machine and human readability and efficiency. While other use cases are possible, the primary CoverageJSON use case is enabling the development of interactive visual websites that display and manipulate environmental data within a web browser.
+Based on JavaScript Object Notation (JSON), CoverageJSON is a format for publishing geo-temporal data to the Web. The primary design goals are simplicity, machine and human readability and efficiency. While other use cases are possible, the primary CoverageJSON use case is enabling the development of interactive visual websites that display and manipulate environmental data within a web browser.
 
-Implementation experience has shown that CoverageJSON is an effective, efficient format, friendly to web and application developers, and therefore consistent with the current https://ogcapi.ogc.org/[OGC API developments]. CoverageJSON supports the efficient transfer, from big data stores, of useable quantities of data to lightweight clients, such as browsers and mobile applications. This enables straightfoward local manipulation of the data, for example, by science researchers. Web developers often use and are familiar with JSON formats.
+Implementation experience has shown that CoverageJSON is an effective, efficient format, friendly to web and application developers, and therefore consistent with the current OGC API developments. CoverageJSON supports the efficient transfer from big data stores of useful quantities of data to lightweight clients, such as browsers and mobile applications. This enables local manipulation of the coverage data in a format familiar to, and popular with, web developers, and readily usable, for example, by science researchers.
```

Comment:
The last sentence has too many clauses

Reply:
Agreed. Also lost the link. I will revert this change.